src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -1085,6 +1089,36 @@ def nparticles_tracked(self) -> int:
         """
         return int(self._tracked_particle_mask.sum())
 
+    @property
+    def _stopped_particle_mask(self) -> NDArray[np.bool_]:
+        """
+        Calculates a boolean mask corresponding to particles that have not been stopped or removed.
+        """
+        # See Class docstring for definition of `stopped` and `removed`
+        return np.logical_and(~np.isnan(self.x[:, 0]), np.isnan(self.v[:, 0]))
+
+    @property
```

Comment:
These new properties match `_tracked_particle_mask` and `nparticles_tracked`, but for stopped and removed particles, by applying the definition of those terms.